templates/home.hamlet

```diff
@@ -19,7 +19,20 @@
     <div .row>
         <div .span12>
             <p>
-                Stackage is a stable source of Haskell packages. We guarantee that packages build consistently and pass tests before generating nightly and Long Term Support (LTS) releases.
+                Stackage is a stable source of Haskell packages. It guarantees that packages build consistently and pass tests before generating nightly and Long Term Support (LTS) releases.
+            <p>
+                A Stackage snapshot includes pinned package versions from <a href="https://hackage.haskell.org">Hackage</a> (the index of all Haskell packages in all versions).
+                It is a curated set of packages that work well together, similar to how a <a href="https://www.debian.org">Debian</a> release is a curated snapshot of most of the open-source software in the world.
+            <p>
+                <a href="https://haskellstack.org">Stack</a> is a tool made specifically to make using Stackage snapshots easy and convenient.
+                We recommend you to use it.
+                But you can also use Stackage in other ways: as a source for packages that work well together, or to ensure that Haskell packages you author or care about work well with others.
+            <p>
+                Stackage is a community project: Haskell users around the world work together to create the stable snapshots.
+                <br />
+                Stackage itself and all tools surrounding it are open-source.
```

Comment:
About this. It makes me wonder. `stackage-nightly` and `stackage-snapshots` have an MIT license. `lts-haskell` has no license (?). Should we additionally release these yaml files under a CC license? I'm not really sure about licensing _data_ vs _software_. But the intent is certainly for all of this stuff to be "open-source".

I'm bringing this up here, because I think we should parenthetically mention, at this point on the website, the specific open-source license(s) that we use.

Reply:
Why CC over MIT? The lack of a license file in `lts-haskell` is certainly just an oversight.

Reply:
For the lack of license in `lts-haskell`, I suggest to open an issue there, I can't really address it as part of this PR.

> I think we should parenthetically mention, at this point on the website, the specific open-source license(s) that we use

@DanBurton Why you think it is important to list the specific licenses here? I was thinking that if the reader is interested in the specific license of each project, they can just go to the project and check; I don't see a big benefit of duplicating this information here. I also think it's unnecessary detail when describing what Stackage is in a few sentences, but happy to be convinced otherwise.

If you think it's important, another alternative might be `... are open-source under permissive licenses`.